ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-multicast.j2

```diff
@@ -4,6 +4,20 @@
 router multicast
 {%     if router_multicast.ipv4 is arista.avd.defined %}
    ipv4
+{%         if router_multicast.ipv4.rpf.routes is arista.avd.defined %}
```

Comment:
This if check can be removed as the natural_sort will take care of that test.

Reply:
thanks!